Automattic/jetpack
# Social: Add publicize package to social plugin

#### Changes proposed in this Pull Request:


- Adds publicize package into the social plugin to allow us to functionally test.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



This is a very ugly at getting the Publicize package to actually run inside the Social plugin. In short, all I did was:

- Require the package for the plugin
- `composer install` and then `composer update` of the plugin
- Load, plugin, notice error due to `jetpack_require_lib()` not being defined...
- Copied over the class that `jetpack_require_lib()` was calling
- Commented out one other line

At this point, the plugin will build with the `publicize` package. Next, we should find out how to get the connections to actually work.

#### Testing instructions:






- Checkout PR
- Build plugin with `jetpack build plugins/social`
- Ensure that wp-admin loads and that you can connect the site
